scripts/bin/base_acl_copp_svc.sh

```diff
@@ -13,8 +13,7 @@
 # See the Apache Version 2.0 License for specific language governing
 # permissions and limitations under the License.
 
-export LD_LIBRARY_PATH=/usr/lib/opx:/usr/lib/x86_64-linux-gnu:/usr/lib:/lib
-export PYTHONPATH=/usr/lib/opx  
+. /etc/opx/opx-environment.sh
 
```

Comment:
I believe the script, "scripts/bin/base_acl_copp_svc.sh", is only called from the .service file. If that's the case the environment is set already from the .service file, and there is no need for this line here. Are we using it else where?